packaging/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ set(Halide_INSTALL_HELPERSDIR "${CMAKE_INSTALL_LIBDIR}/cmake/HalideHelpers"
 set(Halide_INSTALL_PLUGINDIR "${CMAKE_INSTALL_LIBDIR}"
     CACHE STRING "Path to Halide plugins folder")
 
-set(Halide_INSTALL_TOOLSDIR "${CMAKE_INSTALL_DATADIR}/tools"
+set(Halide_INSTALL_TOOLSDIR "${CMAKE_INSTALL_PREFIX}/src/halide"
```

Comment:
I don't see either name as being particularly more or less standard than the other, and `DATADIR` in most installations will be set to `share/[hH]alide`, anyway, so the suffix is unnecessary. I would say to drop this change for the sake of keeping the PR minimal.

Reply:
Oh, also, you should _never_ read `CMAKE_INSTALL_PREFIX` in the CMakeLists.txt. Doing this will break `cmake --install build-dir --prefix <new-prefix>` workflows.